src/components/content-types/workspace/Workspace.tsx

```diff
@@ -81,6 +81,11 @@ export default function Workspace({
       return
     }
 
+    // don't add raw view urls
+    if (parseDocumentLink(currentDocUrl).type === "raw") {
```

Comment:
this violates the "never check a type name" principal

Reply:
I've turned it into a config option in content type